commons/passive/garmin/garmin_generic_heartrate.avsc

```diff
@@ -36,11 +36,13 @@
          "type":"int",
          "doc":"Daily low heart rate (bpm)."
       },
-      {
-         "name":"heartRateSource",
-         "type":"string",
-         "doc":"Heart rate source. Possible values are NO_SOURCE, HR_STRAP, OHR_NO_LOCK, OHR_LOCKED."
-      }
+
+     {"name": "heartRateSource", "type": {
+            "name": "Source",
+            "type": "enum",
+            "doc": "Possible connection state values - UNKNOWN, NO_SOURCE, HR_STRAP, OHR_NO_LOCK, OHR_LOCKED.",
+            "symbols": ["NO_SOURCE", "HR_STRAP", "OHR_NO_LOCK" , "OHR_LOCKED"]
```

Comment:
Please add `"UNKNOWN"` as the first symbol. Do you know what the other values mean? Could you please write a few words for each symbol?

Reply:
Updated. available in latest check-in.